crud/common/schema.lua

```diff
@@ -0,0 +1,107 @@
+local fiber = require('fiber')
+local msgpack = require('msgpack')
+local digest = require('digest')
+local vshard = require('vshard')
+
+local const = require('crud.common.const')
+
+local schema = {}
+
+local function call_reload_schema(replicasets)
+    for _, replicaset in pairs(replicasets) do
+        local master = replicaset.master
+        master.conn:reload_schema()
+    end
+end
```

Comment:
reload_schema calls ping under the hood. It's not async. Seems you could perform such operation in several fibers.